api/src/main/java/net/md_5/bungee/api/event/ResourcePackRequestEvent.java

```diff
@@ -0,0 +1,67 @@
+package net.md_5.bungee.api.event;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.connection.Connection;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+import net.md_5.bungee.api.plugin.Cancellable;
+import net.md_5.bungee.api.plugin.Event;
+
+/**
+ * Called when a server sends a request for a resource pack to
+ * the client
+ */
+@Getter
+@RequiredArgsConstructor
+@ToString
+@EqualsAndHashCode(callSuper = false)
+public class ResourcePackRequestEvent extends Event implements Cancellable
```

Comment:
This is also called with API method. Perhaps a better name is needed

Reply:
all ears for another name